src/rules/no-unsafe-values.js

```diff
@@ -14,7 +14,8 @@ export default {
 		type: /** @type {const} */ ("problem"),
 
 		docs: {
-			description: "Disallow JSON values that are unsafe for interchange",
+			description:
+				"Disallow JSON values that are unsafe for interchange.  This includes strings with unmatched surrogates, numbers that evaluate to Infinity, numbers that evaluate to zero unintentionally, numbers that look like integers but they are too large, and subnormal numbers (see: https://en.wikipedia.org/wiki/Subnormal_number)",
```

Comment:
Descriptions are meant to be one sentence. I think the original description was fine. The rest of these details should go in the README.

Reply:
Since this is going to modify the readme, I'm going to rebase on top of main.  Hopefully the patch isn't large enough that creates a problem.